src/ray/gcs/redis_gcs_client.cc

```diff
@@ -73,7 +73,16 @@ namespace ray {
 
 namespace gcs {
 
-RedisGcsClient::RedisGcsClient(const GcsClientOptions &options) : GcsClient(options) {}
+RedisGcsClient::RedisGcsClient(const GcsClientOptions &options) : GcsClient(options) {
+#if RAY_USE_NEW_GCS
```

Comment:
You can safely remove NEW_GCS stuff, we won't need any of that.

Reply:
Sure. I will remove it in next PR 5515.